scripts/nayduck.py

```diff
@@ -297,8 +301,11 @@ def main():
     else:
         tests = list(read_tests_from_file(pathlib.Path(args.test_file)))
 
+    if args.regex:
+        tests = [test for test in tests if re.match(args.regex, test.strip())]
```

Comment:
Alternatively, `tests = filter(re.compile(args.regex).match, tests)`.

Reply:
Thanks, that's a good idea.